server/settings/dev.py

```diff
@@ -13,6 +13,7 @@
 SQLALCHEMY_TRACK_MODIFICATIONS = False
 SQLALCHEMY_DATABASE_URI = 'sqlite:///../oksqlite.db'
 # SQLALCHEMY_ECHO = True
+COMPRESS_MIMETYPES = ['text/html', 'text/css', 'text/csv', 'text/xml', 'application/json', 'application/javascript']
```

Comment:
Compressing text/csv is not faster than just sending results directly (have to run level 6 gzip first - which makes it slower)